### PR TITLE
feat: Apercu Typography fixes: RadialProgress children; MenuItem sizing

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### PR Checklist

- [x] Related to Abstract designs:
- [x] Related to JIRA ticket: GM-103 GM-104
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### Description

See Storybook deploy preview:
* Slightly reduced font-size in `MenuItem`s per design's request (Yassine) to our 1rem baseline. 
* Added a `children` option for RadialProgress since that's how it's always used in the site.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.4-alpha.6056ed.0<br/>@codecademy/gamut-styles@5.1.1-alpha.6056ed.0<br/>@codecademy/gamut@14.0.4-alpha.6056ed.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f46a92de03e2bce84939745--gamut-preview.netlify.app>https://5f46a92de03e2bce84939745--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f46a92de03e2bce84939745>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f46aad4e03e2be5159390e5--gamut-preview.netlify.app>https://5f46aad4e03e2be5159390e5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f46aad4e03e2be5159390e5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.55d60d.0<br/>@codecademy/gamut-styles@5.1.3-alpha.55d60d.0<br/>@codecademy/gamut@14.0.6-alpha.55d60d.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.e7b813.0<br/>@codecademy/gamut-styles@5.1.3-alpha.e7b813.0<br/>@codecademy/gamut@14.0.6-alpha.e7b813.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f46af1fe03e2b8010938c87--gamut-preview.netlify.app>https://5f46af1fe03e2b8010938c87--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f46af1fe03e2b8010938c87>Deploy Logs</a></p>

File: packages/gamut-styles/utils/variables/_typography.scss [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
-
-$font-family-suisse: "Suisse", "Apercu", -apple-system, BlinkMacSystemFont,
-  "Segoe UI", "Roboto", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
-  "Helvetica Neue", sans-serif;
+// Todo: these should be deleted in a little bit, once everyone's done using them.
+// GM-89
+$font-base: $font-family-base;
+$font-family-apercu: $font-family-base;
+$font-family-headings: $font-family-base;
+$font-family-suisse: $font-family-accent;
```
Comment (JoshuaKGoldberg) [inline packages/gamut-styles/utils/variables/_typography.scss:22]: (this area is cleaned up in the target branch)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.15018f.0<br/>@codecademy/gamut-styles@5.1.3-alpha.15018f.0<br/>@codecademy/gamut@14.0.6-alpha.15018f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f4820c32b9aca140bb5eebd--gamut-preview.netlify.app>https://5f4820c32b9aca140bb5eebd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f4820c32b9aca140bb5eebd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.e43f77.0<br/>@codecademy/gamut-styles@5.1.3-alpha.e43f77.0<br/>@codecademy/gamut@14.0.6-alpha.e43f77.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f4820f41b82221b76e4e1cf--gamut-preview.netlify.app>https://5f4820f41b82221b76e4e1cf--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f4820f41b82221b76e4e1cf>Deploy Logs</a></p>

**Pull request merged**